src/server/graphql/models/Project/projectSchema.js

```diff
@@ -47,6 +48,10 @@ export const Project = new GraphQLObjectType({
         return createdAt;
       }
     },
+    integration: {
```

Comment:
Ah, cool. I was wondering what we were going to do. So after we move completely to Relay, a Project can have a integration interface that will allow for polymorphic representation of different Project card types?